### PR TITLE
#171380743 Deploy the base app to heroku

### PR DESCRIPTION
**Pull request by descholar-ceo**
#### What does this PR ​ do​ ?
It deploys this Application to heroku

#### Description of Task to be completed?
* Add *Procfile* file which contains the web process to start when this app is deployed to heroku
* Add *app.json* file to tell heroku how to create *REVIEW APPS* whenever a new PR is raised
* Add a sample route in *index.js* file to test it the deployment worked

#### How should this be manually tested?
* Click [here](https://stackup2-bar-ch-deploy-uj30pq3.herokuapp.com/) if you see in your browser this message *{"message":"welcome"}* then the deployment is successful or scroll down to this PR's deployment history, and click on `view deployment`

#### Any background context you want to provide?
To know how to use this deployment please read this [README](https://github.com/Stackup-Rwanda/stackup2-barefoot-backend/blob/ch-deploy-base-app-to-heroku-171380743/README.md) file on *Deployments* section

#### What are the relevant pivotal tracker stories?
[#171380743](https://www.pivotaltracker.com/n/projects/2435482)
#### Screenshots (​ if​ appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import morgan from 'morgan';
 
 const app = express();
 
+// middlewares
```
Comment (Muhire-Josue) [inline src/index.js]: Can you please remove this comment?

**Review by Muhire-Josue — APPROVED** [review]
LGTM

File: src/index.js (hunk 1 above)
Comment (descholar-ceo) [reply src/index.js]: done

**Review by the22mastermind — APPROVED** [review]
Looks good. 
Well done my friend.
Keep up the good work!

**Review by mifeille — APPROVED** [review]

**Review by mifeille — APPROVED** [review]

**Review by since9629 — APPROVED** [review]

**Pull request merged**